backend/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 spring.data.mongodb.database=spotorama
 spring.data.mongodb.uri=${MONGO_URL}
-Mapbox_Token=${Mapbox_Token}
\ No newline at end of file
+Mapbox_Token=${Mapbox_Token}
```

Comment:
This is nothing mind blowing, but you might consider writing this in all caps if you like